core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.spec.ts

```diff
@@ -385,6 +385,25 @@ describe('Topic editor tab directive', function() {
     expect(topicPracticeTabSpy).not.toHaveBeenCalled();
   });
 
+  fit('should not call the TopicUpdateService for practice tab if there are less than 10 questions '+
+   'in the topic, no matter what', function () {
```

Comment:
Remember to remote the `f` from `fit`!
```suggestion
  it('should not set the practice tab as displayed if there are less than 10 '+
   'practice questions in the topic', function () {
```